styleguide/components/41-organisms/theme-footer/_theme-footer.scss

```diff
@@ -3,50 +3,156 @@
   @include theme('border-color', 'color-primary--lighten-4', 'theme-footer-border-bottom-color');
 
   width: calc(100% + 2 * #{$gutter-width});
-  margin-top: 4rem;
-  margin-right: -$gutter-width;
-  margin-left: -$gutter-width;
-  padding: 2.7rem 0 2.1rem;
+  margin-right: -1rem;
+  margin-left: -1rem;
+  padding: 0;
   border-bottom: 2px solid;
 
   + .footer {
     margin-top: 0;
   }
 
-  .content-container {
-    display: flex;
-    flex-direction: column;
-    padding: 0 $gutter-width;
+  .contact {
+    .icon-list {
+      margin: 0;
+
+      li:last-of-type {
+        margin-bottom: 0;
+      }
+
+      @include tablet {
+        column-gap: gutter();
+        column-count: 2;
```

Comment:
Does this work on all browsers we support?

Reply:
Yes.